Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Create Python Image
+FROM python:3.8.4-buster as Base
+
+# Install Poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+ENV PATH="${PATH}:/root/.poetry/bin"
+
+# Create Workdir & Copy the Code
+WORKDIR /DevOps-Course-Starter
+COPY . /DevOps-Course-Starter
+
+# Expose the Port
+EXPOSE 5000
+
+FROM base as development
+
+
+RUN poetry install
+ENTRYPOINT [ "poetry", "run", "flask", "run", "--port", "5000" , "--host", "0.0.0.0"]
+
+FROM base as production
+
+ENV FLASK_ENV=production
```

Comment:
Worth noting that this will be overridden by the setting in your `.env` file if one is provided, but is a good default